requirements.txt

```
@@ -13,4 +13,5 @@ fastapi-users[sqlalchemy]==6.1.2
 asyncpg==0.21.0
 pytest==6.1.2
 requests==2.25.0
-fastapi-mail==1.1.5
\ No newline at end of file
+fastapi-mail==1.1.5
+pytest-asyncio==0.20.1
```

```suggestion
pytest-asyncio==0.20.1

```

Add newline at the end of the file.